package/base-files/files/lib/functions/uci-defaults.sh

```diff
@@ -626,6 +626,70 @@ ucidef_set_hostname() {
 	json_select ..
 }
 
+ucidef_set_wireless() {
+	local ssid="$1"
+	local encryption="$2"
+	local key="$3"
+
+	json_select_object wireless
+		json_add_string ssid "$ssid"
+		[ -n "$encryption" ] && json_add_string country "$encryption"
```

Comment:
This should set the encryption type here, not the country code :slightly_smiling_face: 

Sorry I did not spot this earlier when testing on DAP-1320, apparently my old script still lingered somewhere in the cached filesystem build dir, even after deleting from base-files, so that might have slipped into the image :confused: 

`wireless.default_radio0.encryption='none'` was also generated, but the invalid country code must have disabled it.